src/skins/vector/views/molecules/RoomCreate.js

```diff
@@ -17,11 +17,8 @@ limitations under the License.
 'use strict';
 
 var React = require('react');
-var classNames = require('classnames');
 
-//var RoomCreateController = require("../../../../src/controllers/molecules/RoomCreateController");
-
-var MatrixClientPeg = require("../../../../src/MatrixClientPeg");
+//var RoomCreateController = require('matrix-react-sdk/lib/controllers/molecules/RoomCreateController')
```

Comment:
I think this was from people noting that the controllers were empty and so could just be done away with altogether. I guess they may as well go.